Feature/new figure death loc

yenklabs opened the pull request:
The pull request integrates code for deaths by location data in template and core.

yenklabs committed:
adding data_death_loc dataset

yenklabs committed:
merged with upstream

yenklabs committed:
adding dead loc code

yenklabs committed:
cleaned up data death loc

yenklabs committed:
Merge branch 'master' of https://github.com/jeremymoreau/covid19mtl into feature/new-figure-death-loc

yenklabs committed:
integrated death location code

merwok commented on death_loc_code.py:
```diff
@@ -0,0 +1,14 @@
+data_death_loc.drop(data_death_loc.columns[[0]], axis = 1, inplace=True) 
+data_death_loc['Date'] = pd.to_datetime(data_death_loc['Date'])
+
+data_death_loc.to_csv('/home/yenkie/Projects/covid19mtl/venv/c19mtl/data/data_death_loc.csv', index=False)
```
Probably needs relative paths :slightly_smiling_face:

yenklabs replied:
Ha! There is a a death_loc_code.py file that shouldn't be there. I'll remove that file. :) But the core code is using relative path. :D

yenklabs committed:
remove an unwanted file

jeremymoreau committed:
Use unmodified csv from INSPQ as input

Use the unmodified csv as downloaded from the INSPQ to save time with future updates/simplify scrapping. Rename column names for consistency.

jeremymoreau committed:
Add required labels (todo: update es/zh)

jeremymoreau committed:
Change layout for 4th row of figures

4th row will contain two boxes, one for Montreal and one for QC deaths by place of residence.

jeremymoreau committed:
Add Deaths by place of residence (MTL) fig
